app/src/main/java/org/mozilla/focus/state/AppState.kt

```diff
@@ -80,7 +81,7 @@ sealed class Screen {
     /**
      * The application is locked (and requires unlocking).
      */
-    object Locked : Screen()
+    data class Locked(val bundle: Bundle? = null) : Screen()
```

Comment:
Can documentation be added for the new parameter?

Reply:
Done